src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -102,46 +181,18 @@ const ChartsHolder = ({ projections, region, chartId }: ChartsHolderProps) => {
     scrollToRecommendations();
   }, [chartId, metricRefs, isRecommendationsShareUrl]);
 
-  const stats = projection ? projections.getMetricValues() : {};
   const initialFipsList = useMemo(() => {
-    return [projections.primary.fips];
-  }, [projections.primary.fips]);
-
-  const recommendationsIntro = getDynamicIntroCopy(
-    projection,
-    projections.locationName,
-    projections.getMetricValues(),
-  );
-
-  const recommendationsMainContent = getRecommendations(
-    projection,
-    mainContent.recommendations,
-  );
-
-  const recommendsShareUrl = getRecommendationsShareUrl(region);
-
-  const alarmLevel = projections.getAlarmLevel();
-  const recommendsShareQuote = getShareQuote(
-    projections.locationName,
-    alarmLevel,
-  );
+    return projections?.primary?.fips ? [projections.primary.fips] : [];
```

Comment:
I'm not thrilled about how this works with projections possibly undefined, but I didn't see a better way, open to suggestions if you have any.

Reply:
This can be probably reworked by using `region.fipsCode` instead. The Explore component takes a list of FIPS codes and grabs the projections itself, and since here we have the page region already we can use that instead.

Reply:
ah, nice!  I was originally thinking to do that, but then I saw "no, wait, it's a list, and regions only have one FIPS code" and didn't think it through to see we're building a list of one here.